src/contract_wrappers/modules/module_wrapper.ts

```diff
@@ -73,4 +74,26 @@ export default class ModuleWrapper extends ContractWrapper {
   public takeFee = async (params: TakeFeeParams) => {
     return (await this.contract).takeFee.sendTransactionAsync(params.amount, params.txData, params.safetyFactor);
   };
+
+  protected isCallerTheSecurityTokenOwner = async (txData: Partial<TxData> | undefined): Promise<boolean> => {
```

Comment:
Apart from security token owner, are there any other contracts that wrappers might need to know about? Like factory

Reply:
Modules need to know its factory for sure. And there are cases where they need to check a value in the FeatureRegistry too for example.